etl/steps/data/meadow/wb/2022/wb_gender.py

```diff
@@ -0,0 +1,258 @@
+"""WB Gender Meadow step."""
+import re
+import tempfile
+from pathlib import Path
+from typing import Any, List, Tuple
+
+import pandas as pd
+import structlog
+from owid import walden
+from owid.catalog import Dataset, Table, utils
+from owid.datautils.io import decompress_file
+
+from etl.steps.data.converters import convert_walden_metadata
+
+log = structlog.get_logger()
+
+
+class WBGenderMeadowStep:
+    columns_data_rename = {
+        "Country Name": "location",
+        "Indicator Name": "variable",
+        "Indicator Code": "variable_code",
+    }
+    data_filename = "Gender_StatsData.csv"
+    metadata_filename = "Gender_StatsSeries.csv"
+
+    def __init__(self) -> None:
+        log.info("Loading walden dataset...")
+        self.walden_ds = walden.Catalog().find_one("wb", "2022", "wb_gender")
+
+    @property
+    def columns_relevant(self) -> List[str]:
+        """Get list of relevant columns from data file."""
+        return list(self.columns_data_rename.keys()) + [str(i) for i in range(1960, 2022)]
```

Comment:
I'd recommend defining these kinds of parameters as global variables at the top, to be aware that they need to be changed by the next update.

Reply:
makes sense!